app/scripts/app-init.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/unambiguous
+function tryImport(...fileNames) {
+  try {
+    // eslint-disable-next-line
+    importScripts(...fileNames);
+    return true;
+  } catch (e) {
+    console.error(e);
+    return false;
+  }
+}
+
+// eslint-disable-next-line
+self.oninstall = () => {
+  tryImport('./globalthis.js');
+  tryImport('./sentry-install.js');
+  tryImport('./runtime-lavamoat.js');
+  tryImport('./lockdown-more.js');
+  tryImport('./policy-load.js');
+
+  const fileList = [
+    /** FILE NAMES */
```

Comment:
Do we not have some way to know the names of these files yet?

Reply:
These files are generated dynamically during build. They list currently is:

```
tryImport('./common-0.js');
  tryImport('./common-1.js');
  tryImport('./common-2.js');
  tryImport('./common-3.js');
  tryImport('./common-4.js');
  tryImport('./common-5.js');
  tryImport('./common-6.js');
  tryImport('./background-0.js');
  tryImport('./background-1.js');
  tryImport('./background-2.js');
  tryImport('./background-3.js');
  tryImport('./background-4.js');
  tryImport('./background-5.js');
  tryImport('./background-6.js');
  tryImport('./background-7.js');
```

Reply:
Oh does this actually get replaced with the node writeFile method. Got it got it